src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -753,6 +753,9 @@ let load_config_file filename =
 
 let init_from_config_file ?(genesis_dir = Cache_dir.autogen_path) ~logger
     ~may_generate ~proof_level ~genesis_constants (config : Runtime_config.t) =
+  Logger.info logger ~module_:__MODULE__ ~location:__LOC__
```

Comment:
Should we normalise on American spellings? I mean, normalize? :-)

That is, should this be "Initializing ..." ?